### PR TITLE
Fix deleted projects showing up on dashboard

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/8474

## Testing
**URL to test:** 
https://8474-deleted-projects--atd-moped-main.netlify.app/moped/dashboard

**Steps to test:**

1. Add a project with a signal. Check that it's on the signal dashboard. Then remove the signal, it should no longer be on the dashboard. 
2. Delete a project that was previously visible on the dashboard. Should not be showing up on dashboard. 

---
#### Ship list
- [ ] Code reviewed 
- [x] Product manager approved
- [x] ~Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)~

### CODE REVIEW

File: moped-editor/src/queries/signals.js [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
           {
             moped_proj_components: {
               moped_components: { component_name: { _ilike: "signal" } }
-            }
-            status_id: { _neq: 6 }
-          }
-          {
-            moped_proj_components: {
-              moped_components: { component_name: { _ilike: "signal" } }
+              status_id: {_eq: 1}
```
Comment (chiaberry) [inline moped-editor/src/queries/signals.js:21]: these check that the component itself hasnt been archived/deleted

**Review by amenity — APPROVED** [review]
Works beautifully, @chiaberry ! Thank you.

**Review by frankhereford — APPROVED** [review]
Tested - works great. Code reviewed as well.

I made a comment referring to the possibility in the future of getting these magic numbers out via a look-up table to determine what is deleted, but this solution is A+!

🚢 🚢

**Review by mateoclarke — APPROVED** [review]
👍

**Pull request merged**